Luchynska Tetiana/my-todo-list/src/app/todo/todo.component.html

```diff
@@ -0,0 +1,40 @@
+
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            <h2>My todo list</h2>
+            <button class="btn btn-primary" 
+                [disabled]="!allowAddingItem"
+                (click)="addTodo(newTodo.value); newTodo.value=''">Add</button>
```

Comment:
Try to avoid setting a value to some property in the template. Move it into a component.

Reply:
fixed all

Reply:
I meant this line --> (click)="addTodo(newTodo.value); newTodo.value=''">Add</button>
Move this part (newTodo.value=''") into the component.
It isn't a good practice to use more than one event handler in a template (with a semicolon)  and set some value in a template

Reply:
Okey, fixed